rusk-abi/tests/lib.rs

```diff
@@ -23,15 +23,12 @@ use execution_core::{
             PublicKey as SchnorrPublicKey, SecretKey as SchnorrSecretKey,
         },
     },
-    transfer::phoenix::{
-        PublicKey as PhoenixPublicKey, SecretKey as PhoenixSecretKey,
-    },
     BlsScalar, ContractId,
 };
 use ff::Field;
 use rusk_abi::{ContractData, Session, VM};
 
-const POINT_LIMIT: u64 = 0x1000000;
+const POINT_LIMIT: u64 = 0x4000000;
```

Comment:
Quite expensive deserializing this. Any ideas on making this better? Maybe using the "`raw`" serialization?

Reply:
I will look if it could be improved

Reply:
The question is more along the lines of if we should require non-raw in the first place. It can be argued that the user is responsible for whatever they put in the `owner` part of the contract's metadata. If so, offering an expensive function tat verifies their input as opposed to a cheaper one that doesn't might not be what we want.